File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int cur = 1;
        int n = nums.length;

        for (int i=1;i<n;i++) {
            if (nums[i]==nums[i-1])
                continue;
            nums[cur] = nums[i];
            cur +=1;
        }

        return cur;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        cur=1
        n = len(nums)
        for i in range(1,n):
            if nums[i]==nums[i-1]:continue
            nums[cur] = nums[i]
            cur += 1
        return cur